ipaserver/plugins/user.py

```diff
@@ -174,6 +174,27 @@ def check_last_member(user, protected_group_name=u'admins'):
         raise errors.LastMemberError(key=user, label=_(u'group'),
             container=protected_group_name)
 
+
+def is_in_local_idrange(command, uidnumber, options):
+    # Guaranteed, it will be autogenerated by sidgen
```

Comment:
I find the comment misleading. In this case "it" is a SID but the function is only checking that the uidnumber is within an IPA range which only indirectly applies to SID in this case. Someone in the future might use this for something else and the comment could be confusing.

Reply:
You're right, I will handle this outside of the function.